src/components/popup-menu/popup-menu-item.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import injectSheet from 'react-jss';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import styles from './popup-menu.styles';
+
+function PopupMenuItem({ children, topBorder, button, linkTo, onClick, disabled, classes }) {
+  let content;
+  if (disabled) {
+    content = <div className={cn([classes.link, classes.disabledMenuItem])}>{children}</div>;
+  } else if (button) {
+    content = (
+      <button onClick={onClick} className={cn(classes.link, classes.buttonMenuItem)}>
+        {children}
+      </button>
+    );
+  } else {
+    content = (
+      <Link to={linkTo} onClick={onClick} className={classes.link}>
+        {children}
+      </Link>
+    );
+  }
+  return (
+    <li className={classes.item}>
+      {topBorder && <hr className={classes.hr} />}
+      {content}
+    </li>
+  );
+}
+
+PopupMenuItem.propTypes = {
+  classes: PropTypes.object.isRequired,
+  onClick: PropTypes.func,
+  button: PropTypes.bool,
+  linkTo: PropTypes.string,
+  topBorder: PropTypes.bool,
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.string, PropTypes.number]),
+  disabled: PropTypes.bool,
+};
+
+PopupMenuItem.defaultProps = {
+  onClick: () => {},
+  button: false,
+  children: '',
+  linkTo: '#',
+  topBorder: false,
+  className: '',
+  disabled: false,
+};
+
+export default injectSheet(styles)(PopupMenuItem);
```

Comment:
Related to the comment in package.json: Rather than sending in button we might want to give the user the possibility to send in a `node` instead. We recently introduced this concept in the `<Button />` component here: https://github.com/nordnet/nordnet-ui-kit/pull/384/files. What do you think?

Reply:
Not sure I understand, accepted children props are PropTypes.oneOfType([PropTypes.node, PropTypes.string, PropTypes.number])